src/main/java/com/cloudcomputing/cloudkarma/RemainingResources.java

```diff
@@ -0,0 +1,46 @@
+package com.cloudcomputing.cloudkarma;
+
+public class RemainingResources {
+
+	public class FreeSpace {
+
+		int cpu_avaliable;
+		int memory_avaliable;
+
+		@Override
+		public String toString() {
+			return "CPU avaliable:" + cpu_avaliable + "Memory Avaliable: " + memory_avaliable;
+
+		}
+
+	}
+
+	String ec2InstanceId;
+	String containerInstanceArn;
+	int runningTasksCount;
+	FreeSpace freeSpace = new FreeSpace();
+
+	public RemainingResources(String ec2InstanceId, String containerInstanceArn, int runningTasksCount) {
```

Comment:
Please use @Data @AllArgsConstructor @NoArgsConstructor imported through Lombok framework.
Also, no getters and setters.

Reply:
Added @Data and @ToString from lombok. Did not add constructor annotations because 
a) Not all fields are args(freeSpace)
b)There is no argument constructor for RemainingResources.

Or suggest code changes.

Reply:
if you are using setter to set those values, then no need of using constructor.